src/metadata.yaml

```diff
@@ -1,27 +1,28 @@
-name: "keepalived"
-summary: "Failover and monitoring daemon for LVS clusters"
+name: keepalived
+display-name: Keepalived
+summary: Failover and monitoring daemon for LVS clusters
 maintainers:
-  - "Konstantinos Tsakalozos <kos.tsakalozos@canonical.com>"
-  - "Valentin Boucher <valentin.boucher@kontron.com>"
+- "Konstantinos Tsakalozos <kos.tsakalozos@canonical.com>"
+- "Valentin Boucher <valentin.boucher@kontron.com>"
 description: |
   keepalived is used for monitoring real servers within a Linux Virtual
   Server (LVS) cluster.
 docs: https://discourse.charmhub.io/t/keepalived-docs-index/6043
 tags:
-  - "networking"
-series:
-  - noble
-  - jammy
-  - focal
```

Comment:
Removal of the `series` list (replaced by the charmcraft yaml bases list must be dropped to use the `assumes` metadata)